components/BookmarkButton.js

```diff
@@ -11,8 +11,8 @@ export default function BookmarkButton({ isBookmarked, onToggleBookmark }) {
 
 const StyledButton = styled.button`
   position: absolute;
-  top: 10px;
-  right: 10px;
+  top: 1px;
```

Comment:
The use of `top: 1px` may seem unusual and could create slight alignment issues.

```suggestion
  top: 0;
```

_Little UI notes_ ✏️

Even though it is not part of this PR and more of a QA issue, I would like to briefly mention... 

- That hover effects not available on mobile devices and the active state (scaling down and adding a shadow) can also appear glitchy, as it may look like a bug.
- Although  there’s no fixed size for a toggle (bookmark) button, but there are established guidelines for interactive elements. 
  -  For mobile, Google’s Material Design recommends a minimum touch [target size](https://m3.material.io/components/icon-buttons/specs#2b6575b0-cfa3-494e-9509-d7c1949d11f2) of 48x48 px
  - For desktop, buttons can be smaller since mouse clicks allow for more precision. Typically, a 32-40 px height is a good baseline. 
  - The bookmark button should be large enough for easy access, especially if it’s a frequently used feature.